UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs

```diff
@@ -1,40 +1,53 @@
 ﻿using UnityEngine;
+using UnityEngine.Serialization;
 
 ///	<summary>
 ///	Handles sprite syncing between server and clients and contains a custom animator
 ///	</summary>
 public class SpriteHandler : SpriteDataHandler
 {
-	public SpriteRenderer spriteRenderer;
+	[SerializeField]
+	private SpriteRenderer spriteRenderer;
 
-	public int spriteIndex;
+	[SerializeField]
+	private int spriteIndex;
 
-	public int VariantIndex;
+	[SerializeField]
```

Comment:
```suggestion
	[SerializeField] [FormerlySerializedAs("VariantIndex")]
```

Need to add another attribute [FormerlySerializedAs("VariantIndex")] otherwise any components on prefabs which had this value set will lose their setting.